api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3413,4 +3413,28 @@ public void shouldSetTheCorrectSortWeightWhenAddingAnOrderWithANegativePosition(
 		expectedException.expectMessage("Cannot add a member which is out of range of the list");
 		secondSavedOrderGroup.addOrder(newOrderWithInvalidPosition, secondSavedOrderGroup.getOrders().size() + 1);
 	}
+        
+        @Test
```

Comment:
You should be testing `OrderGroup` class and not `OrderService` because the unit of code that has the behavior you are testing is in `OrderGroup` and not `OrderService`, that's the meaning of unit testing.

Reply:
To be more specific, you should be testing the `OrderGroup.addOrder(Order)` method